core-bundle/src/Resources/contao/library/Contao/Automator.php

```diff
@@ -31,9 +31,9 @@ public function __construct()
 	}
 
 	/**
-	 * Purge the search tables
+	 * Purge the search index
 	 */
-	public function purgeSearchTables()
+	public function purgeSearchIndex()
```

Comment:
Isn't the new method deprecated as well, because people should use the `contao.search.indexer` service?

Reply:
We're only using this method to also purge the search cache folder. I could move that logic to the `DefaultIndexer` though. Wdyt?

Reply:
If we do not need the new method then, I am for it. 👍

Reply:
Agree 👍